### PR TITLE
refactor: enable carbon css reset

### PR DESCRIPTION
**Pull request by OmranAbazid**
## Description

See related issue.

## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

related to #23651

### CODE REVIEW

File: optimize/client/src/components/Analysis/TaskAnalysis/OutlierDetailsModal.tsx [hunk 1]
```diff
@@ -39,17 +40,19 @@ export default function OutlierDetailsModal({
...
-          {t('analysis.task.totalFlowNodeInstances', {count: totalCount})}
-          <span>{getOutlierSummary(count, relation)}</span>
-        </p>
-        <h2>{t('analysis.task.detailsModal.durationChart')}</h2>
+        <Stack gap={4}>
+          <p className="description">
+            {t('analysis.task.totalFlowNodeInstances', {count: totalCount})}
+            <span>{getOutlierSummary(count, relation)}</span>
+          </p>
+          <h4 className="subTitle">{t('analysis.task.detailsModal.durationChart')}</h4>
```
Comment (vitorwtavares) [inline optimize/client/src/components/Analysis/TaskAnalysis/OutlierDetailsModal.tsx:48]: Is this class `subTitle` a global one? I don't see it on the associated `.scss` file.

File: optimize/client/src/components/Reports/controlPanels/DefinitionList/RenameVariablesModal.scss [hunk 2]
```diff
@@ -1,6 +1,9 @@
+@use '@carbon/styles/scss/spacing' as *;
+
 .RenameVariablesModal {
-  .cds--modal-container {
+  .ModalContainer {
```
Comment (vitorwtavares) [inline optimize/client/src/components/Reports/controlPanels/DefinitionList/RenameVariablesModal.scss:4]: Is there a reason for the difference casing on this class name? If not I would suggest you match the other ones using camel case by renaming it to `modalContainer`.

File: optimize/client/src/components/Reports/controlPanels/DefinitionList/RenameVariablesModal.tsx [hunk 3]
```diff
@@ -100,9 +101,9 @@ export default function RenameVariablesModal({
   return (
     <Modal open={open} size="lg" onClose={onClose} className="RenameVariablesModal">
       <Modal.Header title={t('report.definition.variables.rename')} />
-      <Modal.Content>
+      <Modal.Content className="ModalContainer">
```
Comment (vitorwtavares) [inline optimize/client/src/components/Reports/controlPanels/DefinitionList/RenameVariablesModal.tsx:104]: Same thing about the casing of this class.

**Review by vitorwtavares — APPROVED** [review]

File: optimize/client/src/components/Reports/controlPanels/DefinitionList/RenameVariablesModal.scss (hunk 2 above)
Comment (OmranAbazid) [reply optimize/client/src/components/Reports/controlPanels/DefinitionList/RenameVariablesModal.scss:4]: Good catch!
I think i did not by mistake

**Pull request merged**